content/docs/reference-test-renderer.md

```diff
@@ -94,157 +94,157 @@ expect(testInstance.findByProps({className: "sub"}).children).toEqual(['Sub']);
 * [`testInstance.parent`](#testinstanceparent)
 * [`testInstance.children`](#testinstancechildren)
 
-## Reference {#reference}
+## Dokumentacja {#reference}
 
 ### `TestRenderer.create()` {#testrenderercreate}
 
 ```javascript
 TestRenderer.create(element, options);
 ```
 
-Create a `TestRenderer` instance with the passed React element. It doesn't use the real DOM, but it still fully renders the component tree into memory so you can make assertions about it. The returned instance has the following methods and properties.
+Tworzy instancję `TestRenderer` przy użyciu przekazanego elementu reactowego. Nie korzysta z prawdziwego drzewa DOM, lecz renderuje całe drzewo komponentów do pamięci, aby można było wykonać na nim sprawdzenia. Zwracana instancja posiada następujące metody i właściwości:
```

Comment:
Something to consider: https://sjp.pwn.pl/sjp/asercja;2441506.html

Reply:
Fakt, myślałem, że to kalka językowa, a tu proszę - nawet [na Wikipedii jest o tym artykuł](https://pl.wikipedia.org/wiki/Asercja_(informatyka)) :-)